core/src/main/java/hudson/scheduler/CronTab.java

```diff
@@ -532,4 +532,17 @@ private String toString(String key, long bit) {
             return null;
         }
     }
+
+    /**
+     * Returns the configured time zone, or null if none is configured
+     *
+     * @return the configured time zone, or null if none is configured
+     * @since TODO
+     */
+    @CheckForNull public TimeZone getTimeZone() {
```

Comment:
If the fix is a subject for backporting, better to mark this API method as restricted for now